### ISSUE TITLE
Discussion: Branch Cleanup

### ISSUE THREAD

**Issue by crcrewso** [pymedphys/pymedphys#1711]
I see that there are duplications of tags and branches for some of our older versions. Is there any plan to delete any of these branches? The list is getting quite long making it a bit more work when using command line to navigate open feature development branches.

**Comment by SimonBiggs** [comment]
Fair point. If we already have a tag, it is reasonable that we delete the branch if it is not serving any other purpose.

**Comment by crcrewso** [comment]
If the group is okay with it I can confirm there's a tag for every branch and then clean up the duplicates.

**Comment by crcrewso** [comment]
Hearing no objections I'll do the work this weekend.

**Comment by SimonBiggs** [comment]
> Hearing no objections I'll do the work this weekend.

Sounds good to me. I may need to give you higher permissions than you have. Let me know if that's the case.

**Comment by crcrewso** [comment]
> > Hearing no objections I'll do the work this weekend.
> 
> Sounds good to me. I may need to give you higher permissions than you have. Let me know if that's the case.

Good call, I just checked and I can't delete any of the protected branches. 
Some of them are obvious having patches only changing the version names. I'll leave the rest of them alone but will document the issues here.

**Comment by crcrewso** [comment]
First pass Summary of unmerged changes 
0.9.x - 0.38.x -  Version bumps - can be deleted 
0.6.x, 0.7.x and 0.8.x have minor changes requiring due diligence 

I currently don't have the rights to delete these stale branches so I will leave the next steps in the hands of administrators
